backend/src/services/linear/api/resolvers.py

```
@@ -11000,6 +11189,150 @@ def resolve_issueSubscribe(obj, info, **kwargs):
+
+    # Build base query
+    base_query = session.query(IssueLabel)
+
+    # Apply archived filter
+    if not includeArchived:
+        base_query = base_query.filter(IssueLabel.archivedAt.is_(None))
+
+    # Apply additional filters if provided
+    if filter:
+        # Apply team filter
+        if "team" in filter and filter["team"]:
```

Ruff (0.14.4)

Unnecessary key check before dictionary access

Replace with `dict.get`

(RUF019)